template/template_example.Rmd

```diff
@@ -0,0 +1,206 @@
+
+<img src="screenshots/analysis-folder-structure.png" width=400>
+
+In order for our example here to run without a hitch, we need these files to be in these locations so we've constructed a test to check before we get started with the analysis. 
+Run this chunk to double check that your files are in the right place. 
+
+```{r}
+# Check if the gene expression matrix file is in the data directory stored as `data_dir`
+file.exists(file.path(data_dir, <<DATA_ACCESSION FILENAME>>))
+
+# Check if the metadata file is in the data directory stored as `data_dir`
+file.exists(file.path(data_dir, <<METADATA_ACCESSION FILENAME>>))
+```
+
+If the chunk above printed out `FALSE` to either of those tests, you won't be able to run this analysis _as is_ until those files are in the appropriate place.
+
+If the concept of a "file path" is unfamiliar to you; we recommend taking a look at our [section about file paths](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#an-important-note-about-file-paths-and-Rmds). 
+
+***
+
+# <<NAME OF example>>
+
+## Install libraries
+
+See our getting started with [package installation](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#what-you-need-to-install) for a list of the other software you will need, as well as more tips and resources. 
+
+In this analysis, we will be using <<PACKAGE>> [<<CITATION>>].
+
+```{r}
+if (!(<<PACKAGE>> %in% installed.packages())) {
+  # Install this package if it isn't installed yet
+  BiocManager::install(<<PACKAGE>>, update = FALSE)
+}
+```
+
+Attach the <<PACKAGE>>:
+
+```{r}
+# Attach the library
+library(<<PACKAGE>>)
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+<<SET THE SEED IF YOU NEED>>
+```
+
+## Import and set up data
+
+Data downloaded from refine.bio include a metadata tab separated values ("tsv") file and a data tsv file. This chunk of code will read the both tsv files and add them as data.frames to your environment.
+
+```{r Import data from .tsv files}
+# Read in metadata tsv file
+metadata <- readr::read_tsv(file.path(data_dir, # Replace with path to your metadata file
+                                      <<METADATA_ACCESSION FILENAME>> # Replace with the name of your metadata file
+                                      ))
+
+# Read in data tsv file
+df <- readr::read_tsv(file.path(data_dir, # Replace with path to your data file
+                                <<DATA_ACCESSION FILENAME>> # Replace with the name of your data file
+                                )) %>%
+  tibble::column_to_rownames('Gene')
+```
+
+Let's ensure that the metadata and data are in the same sample order. 
+
+```{r}
+# Make the data in the order of the metadata
+df <- df %>% dplyr::select(metadata$geo_accession)
+
+# Check if this is in the same order
+all.equal(colnames(df), metadata$geo_accession)
+```
+
+
+<<<< ALL THE ANALYSIS STEPS HERE>>>>>>>>
+
+
+# Using your own data with this analysis
+
```

Comment:
`Using a different refine.bio dataset with this analysis`?

I think `own data` makes it sound like they can plug in non-refine.bio datasets. (which they can but not without significant changes, which we don't really get into) 

Also I want to suggest moving this section before `Install libraries`.

Reply:
Ah! Good point. Will change!

Reply:
What about the `Further learning resources about this analysis` section? Should that also move toward the top or stay at the end?